src/formio.alerts.component.ts

```diff
@@ -8,5 +8,7 @@ import { FormioAlerts } from './formio.alerts';
 })
 export class FormioAlertsComponent {
   @Input() alerts: FormioAlerts;
-  constructor() {}
+  constructor() {
+    this.alerts = new FormioAlerts();
```

Comment:
We should not be creating new instances of alerts here. The components manage the alerts instance and then pass that instance into this component using ```[alerts]="alerts"```. I think the fix here is to just add that to the FormioGrid component template.